### PR TITLE
docs: Create responsive apps

### PR DESCRIPTION
**Pull request by jeromecambon**
Covers [BPM-324](https://bonitasoft.atlassian.net/browse/BPM-324)

[BPM-324]: https://bonitasoft.atlassian.net/browse/BPM-324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/applications/pages/ui-builder/create-fully-responsive-apps.adoc [hunk 1]
```diff
@@ -0,0 +1,36 @@
+= Create Fully Responsive Applications
+:page-aliases: applications:create-fully-responsive-apps.adoc
```
Comment (benjaminParisel) [inline modules/applications/pages/ui-builder/create-fully-responsive-apps.adoc]:
```suggestion
```

:page-aliases: is useless when the page didn't exist before. remove it please

**Review by benjaminParisel — CHANGES_REQUESTED** [review]

**Review by vhemery — COMMENTED** [review]
Nitpick:
Can we align the page, menu and card title ?

- Create Fully Responsive Applications
- Ensure the application is fully responsive
- Make your application responsive

File: modules/applications/applications-nav.adoc [hunk 2]
```diff
@@ -31,6 +31,7 @@
     ***** xref:ui-builder/common-apis-to-use.adoc[Common APIs to use]
     ***** xref:ui-builder/how-to-retrieve-user-avatar.adoc[Retrieve user avatar image]
     ***** xref:ui-builder/reuse-code-and-components.adoc[Reuse code and components]
+    ***** xref:ui-builder/create-fully-responsive-apps.adoc[Ensure the application is fully responsive]
```
Comment (laurentleseigneur) [inline modules/applications/applications-nav.adoc]: we can challenge `fully`

File: modules/applications/pages/ui-builder/bonita-ui-builder-best-practices.adoc [hunk 3]
```diff
@@ -38,3 +38,8 @@ xref:ui-builder/how-to-retrieve-user-avatar.adoc[[.card-title]#Retrieve user ava
 --
 xref:ui-builder/reuse-code-and-components.adoc[[.card-title]#Reuse code and components# [.card-body.card-content-overflow]#pass:q[Improve maintenance and consistency between your applications]#]
 --
+
+[.card.card-index]
+--
+xref:ui-builder/create-fully-responsive-apps.adoc[[.card-title]#Make your application responsive# [.card-body.card-content-overflow]#pass:q[Enhance the layout to ensure the application is fully responsive]#]
```
Comment (laurentleseigneur) [inline modules/applications/pages/ui-builder/bonita-ui-builder-best-practices.adoc]: same comment for `fully`

**Review by marcobonita — APPROVED** [review]

**Comment by laurentleseigneur** [comment]
LGTM

File: modules/applications/applications-nav.adoc (hunk 2 above)
Comment (jeromecambon) [reply modules/applications/applications-nav.adoc]: I would be agree with you, but when changing that, I see that one can have the feeling that the app built with Appsmith is not responsive by default... Which is not the case. So finally I prefer to keep it like it is.

**Review by benjaminParisel — APPROVED** [review]

**Review by vhemery — APPROVED** [review]

**Pull request merged**